modules/ROOT/pages/edifact/edifact-edi-connector.adoc

```diff
@@ -276,6 +278,7 @@ Settings for writing messages:
 * Enforce valid characters for UNOA and UNOB syntax levels.
 * Request acknowledgments for send interchanges using Acknowledgment Request field (UNB.9).
 * Set a test indicator digit to be used on send interchanges (UNB.11).
+* Whether to use the default output MIME type `application/plain`, or the Edifact-specific alternative `application/edifact`.
```

Comment:
I think Kim would have us remove the comma, but if you feel it clarifies the text, leave it in.

Reply:
Fixed.